tests/group_by_test.py

```diff
@@ -136,6 +136,24 @@ def test_group_by_median(constructor: Constructor) -> None:
     assert_equal_data(result, expected)
 
 
+def test_group_by_std(constructor: Constructor) -> None:
+    data = {"a": [1, 1, 2, 2], "b": [5, 4, 3, 2]}
+    result = (
+        nw.from_native(constructor(data)).group_by("a").agg(nw.col("b").std()).sort("a")
+    )
+    expected = {"a": [1, 2], "b": [0.707107] * 2}
+    assert_equal_data(result, expected)
+
+
+def test_group_by_var(constructor: Constructor) -> None:
+    data = {"a": [1, 1, 2, 2], "b": [5, 4, 3, 2]}
+    result = (
+        nw.from_native(constructor(data)).group_by("a").agg(nw.col("b").var()).sort("a")
+    )
+    expected = {"a": [1, 2], "b": [0.5] * 2}
+    assert_equal_data(result, expected)
```

Comment:
a very tiny comment from me as well :)
we've recently added a test above (`test_group_by_depth_1_agg`) which tries to keep this "family" of tests together; perhaps, we might add the `var` test there (the `std` test is already present instead)

Reply:
Didn't notice this. Was wondering where did test with `std` gone! Done changes